main.go

```diff
@@ -1,14 +1,21 @@
 package main
 
 import (
+	log "github.com/Sirupsen/logrus"
 	"github.com/rancherio/go-machine-service/events"
 	"github.com/rancherio/go-machine-service/handlers"
-	"log"
 	"os"
 )
 
 func main() {
-	log.Println("Beginning go-machine-service...")
+	// Process options
+	for _, f := range os.Args {
```

Comment:
Two things:

1. This is good enough for now, but eventually we'll probably want to support --log-level=[debug|info|warn|error]. Would it be much  more effort for you to change it to that?
2. Even if you don't do number 1, can you extract this to a function called `setLogLevel` or something like that?